index.html

```diff
@@ -0,0 +1,894 @@
+                                        alt="Winter Landscape" class="w-100" />
+                                </div>
+                                <div class="img-fluid"  class="multi-carousel-item">
+                                    <img src="https://mdbcdn.b-cdn.net/img/Photos/Thumbnails/Slides/3.webp"
+                                        alt="View of the City in the Mountains" class="w-100" />
+                                </div>
+                                <div class="multi-carousel-item">
+                                    <img class="img-fluid" src="https://mdbcdn.b-cdn.net/img/Photos/Thumbnails/Slides/4.webp"
+                                        alt="Place Royale Bruxelles" class="w-100" />
+                                </div>
+
+                            </div>
+                        </div>
+
+                    </div>
+                    <div class="col-lg-5" height="100%" style="margin-left: 5%;">
+                        <div class="font-weight-bold" style="font-size: 25px; color: #192BB9;">
+                            <span>AGELESS MULTI-COLLAGEN® PROTEIN POWDER</span>
+                        </div>
+                        <div>
+                            <img class="img-fluid" src="assets/p1.webp" alt="">
+                            <img class="img-fluid" style="padding-top: 100px;" src="assets/asdf.avif" alt="">
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="col-6 text-center pt-5  ">
+                <div class="container">
+                    <span class="font-weight-bold" style="font-size:x-large ; color: blue;">Choose Your Pakage</span>
+                </div>
+                <span>Select your money-saving option below:</span>
+                <div class="row text-center pt-3" style="margin-right:0%; margin-left: 5%;">
+                    <div class="col-5" height="30px"
+                        style="overflow: hidden; border: 2px solid blue; color: white; margin-right: 2%; border-radius: 0.25rem; font-size: 25px; background-color: #192BB9;">
+                        One Time Purchase
+                    </div>
+                    <div class="col-5"
+                        style="overflow: hidden; border: 2px solid blue; color: blue; margin-right: 2%; border-radius: 0.25rem; font-size: 25px;">
+                        Subscribe & Save 36%
+                    </div>
+                </div>
+                <div class="col-6 text-center"></div>
+                <div class="row pt-3" style="margin-right: 5%;">
+                    <div class="col-md-4 font-weight-bold mb-3 mt-3"
+                        style="border: 2px solid darkblue; font-size: 30px;">
+                        <div class="pt-3" style="color:blue">
+                            <span>1 Bag</span>
+                        </div>
+                        <div class="pt-3" style="padding-bottom: 120px;">
```

Comment:
avoid inline styles in this whole file and try to use a relative unit like rem for widths, margins, padding, border sizes, and font sizes, it will help you to achieve responsiveness.